src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/Repository.kt

```diff
@@ -260,8 +261,9 @@ class Repository(
     relationshipTypeRepository.findByIdOrNull(Pk(RelationshipType.RELATIONSHIP, code))!!
 
   fun lookupAgency(id: String): AgencyLocation = agencyLocationRepository.findByIdOrNull(id)!!
-  fun lookupAgencyInternalLocationByDescription(description: String): AgencyInternalLocation =
-    agencyInternalLocationRepository.findOneByDescription(description).map { it }.orElse(null)
+  fun lookupAgencyInternalLocationByDescription(description: String): AgencyInternalLocation? =
+    agencyInternalLocationRepository.findOneByDescription(description).getOrNull()
+      ?.also { it.toString() } // hydrate
```

Comment:
Hydration was needed in the new test

Reply:
BTW @steverendell - you shouldn't really need to this in your tests when the assets run  in a `runInTransaction` block